app.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const app = express()
+//Will need to pull in error handler here
+
+const { getTopics } = require('./Controller/topics.controller')
+
+//app.use(express.json()) will go here
```

Comment:
okay, this is fine! Would ask you to remove if it wasn't commented out.  But think you understand what it's purpose is!